src/helpers/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
buchu doesn't use 'index.js' for its file structure. right or wrong, lets keep the project consistent.

Reply:
Well, I had a consistent code in mind when building this structure, coz it prevents the invocation of many unnecessary imports. I mean:

instead we use this:

import helpFunc1 from './helpers/helpFunc1'
import helpFunc2 from './helpers/helpFunc2'
import helpFunc2 from './helpers/helpFunc3'

We can do it:

import { helpFunc1, helpFunc2, helpFunc3 } from './helpers' 

But ok, i will change.

Reply:
Well, when i was programming this structure, I had a better readable code in mind, coz it prevents the invocation of many unnecessary imports. I mean:

Instead we use this:

const helpFunc1 = require('./helpers/helpFunc1')
const helpFunc2 = require('./helpers/helpFunc2')
const helpFunc2 = require('./helpers/helpFunc3')

We can do it:

const { helpFunc1, helpFunc2, helpFunc3 } = require('./helpers')

But ok, i will change.